cmd/juju/commands/common.go

```diff
@@ -112,19 +112,19 @@ func environFromNameProductionFunc(
 	return env, cleanup, err
 }
 
-// resolveCharmURL resolves the given charm URL string
+// resolveEntityURL resolves the given charm or bundle URL string
```

Comment:
There's an unfortunate conflict of naming between the charm store and juju-core. In the charm store, an "entity" refers to a charm or a bundle, but in juju-core, it refers to something with a Tag method (a unit, machine, user etc).

Although this method name would be fine in charmstore context, I suspect it will be confusing when read in the context of juju-core. I'm thinking that instead of "entity" we should use "charmOrBundle" instead. Alternatively "charmStoreEntity" might work (which has the advantage that it will still be accurate if we add another kind of thing to the charm store).

What do you think?